### PR TITLE
Added LF442-MILACN Device

### PR DESCRIPTION
**Pull request by pinoaffe**

### CODE REVIEW

File: dev/807cb705-5a00-47fa-b2a3-63aca7415820/device.lp [hunk 1]
```diff
@@ -0,0 +1,21 @@
+(librepcb_device 807cb705-5a00-47fa-b2a3-63aca7415820
+ (name "LF442-MILACN")
+ (description "LF442-MIL Dual Low Power JFET Input Operational Amplifier With PDIP Footprint")
+ (keywords "")
```
Comment (dbrgn) [inline dev/807cb705-5a00-47fa-b2a3-63aca7415820/device.lp]: Maybe you could add a few keywords? For example `lf442,pdip,jfet,amplifier`. Maybe you can also think of others.

**Comment by pinoaffe** [comment]
Sure, I guess the TL074xN should also be moved to that library, then?

**Comment by dbrgn** [comment]
Probably, but I don't know how LibrePCB handles moving items between libraries... @ubruhin probably knows more.

**Comment by ubruhin** [comment]
Yep, would be good to move the device to the new TI library :)

**Pull request closed without merging**